exercises/reverse-string/metadata.yml

```diff
@@ -1,5 +1,5 @@
 ---
 blurb: "Reverse a string"
-title: "reverse-string"
+title: "Reverse String"
 source: "Introductory challenge to reverse an input string"
-source_url: "https://medium.freecodecamp.org/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb"
\ No newline at end of file
+source_url: "https://medium.freecodecamp.org/how-to-reverse-a-string-in-javascript-in-3-different-ways-75e4763c68cb"
```

Comment:
unrelated whitespace change should go in its own commit in this PR, or in a different PR

Reply:
Sorry about that, I'm not bothered by the newline missing so I'll leave it as it is.